ads/google/a4a/utils.js

```diff
@@ -229,6 +229,8 @@ export function googlePageParameters(win, nodeOrDoc, startTime) {
           'ish': win != win.top ? viewportSize.height : null,
           'art': art == '0' ? null : art,
           'vis': visibilityStateCodes[visibilityState] || '0',
+          'debug_experiment_id': (getMode().localDev &&
```

Comment:
Are you sure this doesn't wind up setting the value to `false` in production? I'm not sure what happens in that case. I generally set to an explicit null rather than using boolean operators.

Reply:
I wound up removing the local dev constraint